go/vt/servenv/grpc_server.go

```diff
@@ -135,6 +138,9 @@ func createGRPCServer() {
 
 		// create the creds server options
 		creds := credentials.NewTLS(config)
+		if *GRPCEnableOptionalTLS {
```

Comment:
Can you log a message here at the warning level to indicate that plan-text connections will be accepted.